### ISSUE TITLE
wrong package-lock.json in Angular MoviesApp

### ISSUE THREAD

**Issue by andreatosato** [dotnet/tye#1165]
tye/samples/apps-with-core-angular/MoviesApp/package-lock.json has many error

https://kdi.jfrog.io/kdi/api/npm/npm is under authentication:

**http fetch GET 401** https://kdi.jfrog.io/kdi/api/npm/npm/jasmine-core/-/jasmine-core-2.8.0.tgz

How can I build correctly project?

**Comment by andreatosato** [comment]
I change Dockerfile to force package.lock.json update

**npm install --force**

```yaml
FROM node:alpine as node

WORKDIR /src
COPY . .

RUN npm install --force && npm run build

FROM nginx:alpine
COPY nginx.conf /etc/nginx/nginx.conf
COPY --from=node /src/dist /usr/share/nginx/html/movies
```

**Comment by pratiksanglikar** [comment]
Hi @andreatosato, I believe this issue is fixed with #1162.
Can you please give it a try again and reopen if it still reproduces.
